client/pages/Puzzle1Page/Puzzle1Page.jsx

```diff
@@ -177,7 +196,14 @@ export default function Puzzle1Page() {
                     </Modal.Title>
                   </Modal.Header>
                   <Modal.Body>
-                    You have completed Puzzle 1!
+                    <Form.Group>
+                      {/* If stoppedTime is not zero, display the message with the time */}
+                      {stoppedTime > 0 && (
+                        <Form.Label>
+                            You have completed Puzzle 1 in {minutes}:{seconds?.toString().padStart(2, '0') || "00"}!
+                        </Form.Label>
+                      )}
+                    </Form.Group>
```

Comment:
Again, confused on why this is a form.

Additionally, we should probably be showing this in mm:ss.sss, since the database uses milliseconds to store time.